yarn.lock

```diff
@@ -2532,7 +2532,7 @@
   dependencies:
     stackframe "^1.1.1"
 
-"@discoveryjs/json-ext@0.5.7", "@discoveryjs/json-ext@^0.5.0":
```

Comment:
Not sure why these were in here before but they are not used at all and this time yarn stripped them out.